packages/components/src/components/date-picker/date-picker.scss

```diff
@@ -113,7 +113,6 @@
 
             &-text {
                 font-size: 1.4rem;
-                color: var(--text-prominent);
```

Comment:
I think removing this will break dark theme

Reply:
Let me check it first

Reply:
I removed an extra border and now date picker looks fine on both themes, these styles just overwrite our input styles and break it for mobile disabled datepickers